### PR TITLE
Add administrate dashboard for user management

### PR DESCRIPTION
**Pull request by jfranciscosousa**
The title. Only setup this for users and nothing else right now.

After logging in **moderators (only)** can go to /admin and access the dashboard.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -12,4 +12,9 @@ class User < ApplicationRecord
 
   has_many :book_presentations, dependent: :destroy
   has_many :books, -> { distinct }, through: :book_presentations
+
+  enum role: {
+    user: "user",
+    admin: "admin"
+  }
 end
```
Comment (jpcorreia99) [inline app/models/user.rb:20]: Is this addition demanded by the administrate gem or can the already existing moderator flag be used instead?

**Comment by gilafonso** [comment]
Run the linters please

File: app/models/user.rb (hunk 1 above)
Comment (jfranciscosousa) [reply app/models/user.rb:20]: Ignore this I'll delete

**Review by gilafonso — APPROVED** [review]
👌

**Review by jpcorreia99 — APPROVED** [review]
:shipit:

**Pull request merged**